chain/src/main/scala/org/bitcoins/chain/config/ChainAppConfig.scala

```diff
@@ -98,6 +98,10 @@ case class ChainAppConfig(
   lazy val filterBatchSize: Int =
     config.getInt(s"${moduleName}.neutrino.filter-batch-size")
 
+  lazy val forceRecalcChainWork: Boolean =
+    config.getBooleanOrElse(s"$moduleName.neutrino.filter-batch-size",
```

Comment:
You didn't change `filter-batch-size` in the string

Reply:
Also we should probably at least add it to the example configurations here otherwise it is soooo easy to forget.